site2/docs/io-http-sink.md

```diff
@@ -0,0 +1,56 @@
+---
+id: io-http-sink
+title: HTTTP sink connector
+sidebar_label: "HTTP sink connector"
```

Comment:
I think we don't show this page on the sidebar so this metadata is unnecessary. But it's not a blocker.

Reply:
Then it should be removed from all the connector pages I guess ?

Reply:
Yep. I comment here, and possibly we file an issue to track such a movement :)